src/com/android/launcher3/allapps/ActivityAllAppsContainerView.java

```diff
@@ -478,6 +478,9 @@ public void reset(boolean animate) {
      *                   apps list.
      */
     public void reset(boolean animate, boolean exitSearch) {
+        
+        boolean showFastScoller = PreferenceExtensionsKt.firstBlocking(pref2.getShowScrollbar());
+        mFastScroller.setVisibility(showFastScoller ? VISIBLE : INVISIBLE);
```

Comment:
Please move this in `initContent` so changes will take effect immedietly

Reply:
Does the launcher not have to be restarted for changes to apply then?

Reply:
Yes no need just change in pref onSet = { reloadHelper.recreate() }

Reply:
Fixed